packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1008,6 +1008,7 @@ class ListView extends BoxScrollView {
   /// child that could possibly be displayed in the list view instead of just
   /// those children that are actually visible.
   ///
+  /// Note that this constructor does not dynamically change the [List] once generated.
```

Comment:
style nit: avoid "note that", see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose